### DISCUSSION TITLE
rest Api

### DISCUSSION THREAD

**Discussion by cm4rs** [music-assistant/support#2735] [Q&A] [answered]
Hi everybody,

At home I use mpd + snapcast to broadcast music. I have a Spotify and a Qobuz account and I think music-assistant would be perfect to unify my flacs, spotify, qobuz and radio streaming.
Over the year I wrote several scripts to change radio by crontab or to broadcast what's streaming, title, artist and cover but as I often listen to Radio France station I have a python scripts which is getting json on their api to know what's diffused.

I installed music-assistant docker image and it seems very promising, but I did not find anywhere methods to :
 - change program via curl or shell / python script
 - subscribe to status change

PS: English is not my native language so sorry if I'm not clear

**Comment by OzGav** [comment, marked as answer]
Music Assistant has tight integration with Home Assistant so all control is done with automations from there.

**Reply by cm4rs** [reply]
Ok! Anyway I wanted to try home-assistant for a long time.
Thanks!
